ml-agents/tests/yamato/scripts/run_compressed_sensor.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import numpy as np
+
+from mlagents_envs.environment import UnityEnvironment
+
+EPSILON = 0.001
+
+
+def test_run_environment(env_name):
+    """
+    Run the low-level API test of compressed sensors using the specified environment
+    :param env_name: Name of the Unity environment binary to launch
+    """
+    env = UnityEnvironment(
+        file_name=env_name, no_graphics=True, additional_args=["-logFile", "-"]
+    )
+
+    try:
+        # Reset the environment
+        env.reset()
+
+        env.step()
```

Comment:
Why multiple steps? Maybe set the DecisionRequester period to 1?

Reply:
The decision requester is already set to 1. I wanted the stacking to be complete